cypress/plugins/index.js

```diff
@@ -0,0 +1,21 @@
+require('../../src/server/babelRegister')
+const wp = require('@cypress/webpack-preprocessor')
+const webpackOptions = require('./webpack.cypress.config')
+const dotenv = require('dotenv')
+const resetDb = require('./resetDb')
+
+module.exports = (on, config) => {
+  const options = {
+    webpackOptions,
+    watchOptions: {}
+  }
+  const path = process.env.CI ? '.env' : '.env.test'
+  config.env = dotenv.config({silent: true, path})
+  on('file:preprocessor', wp(options))
+  const dbOptions = {source: 'cypress', target: 'test'}
+  on('task', {
```

Comment:
What is the task event? I feel this could use a comment.

Reply:
it's not implemented yet, as soon as we start testing mutations i'll build it out.
basically it's a wait to run node.js code from a test.
so for example before each test we can reset the DB